jobs/config.json

```diff
@@ -10834,6 +10834,12 @@
       "sig-instrumentation"
     ]
   },
+  "pull-heapster-e2e-prow": {
+    "scenario": "execute",
```

Comment:
assume you'd need some args to execute here?

Reply:
Isn't this just executing the k8s container specified in the other file?

Reply:
the previous job uses `"scenario": "kubernetes_heapster",` though

Reply:
True, but kubernetes_heapster scenario runs a container via docker, while I wanted the prow version to run the same container in a k8s pod.

Reply:
ohhhh I see, the current image does not even checks out the heapster repo :-) I'll give it a patch